src/rules/memberAccessRule.ts

```diff
@@ -159,6 +171,8 @@ function typeToString(node: ts.ClassElement): string {
             return "get property accessor";
         case ts.SyntaxKind.SetAccessor:
             return "set property accessor";
+        case ts.SyntaxKind.Parameter:
+            return "class parameter property";
```

Comment:
I think you can omit "class" since parameter properties can only occur in classes.